tests/fixtures/config-hierarchy/overrides/.eslintrc

```diff
@@ -0,0 +1,38 @@
+{
+    "rules": {
+        "quotes": [2, "double"]
+    },
+    "overrides": [
+        {
+            "files": "foo.js",
+            "rules": {
+                "quotes": [2, "single"]
```

Comment:
Nitpick: Can you use different rules for the different overrides tests here rather than using `quotes` for all of them? If there is a bug where some overrides get applied that shouldn't be, I want to make sure the bug is caught by the tests.